examples/Microsoft.Spark.FSharp.Examples/Program.fs

```diff
@@ -5,23 +5,18 @@
 module Microsoft.Spark.Examples.Main
 
 open System
-open System.Collections.Generic
-open System.Linq
 open System.Reflection
-open System.Runtime.InteropServices
 
-let printUsage (examples : IEnumerable<string>) =
+let printUsage (examples : string seq) =
```

Comment:
Not a dealbreaker by any means, but `seq<string>` is preferred here as per the style guide: https://docs.microsoft.com/en-us/dotnet/fsharp/style-guide/formatting#use-prefix-syntax-for-generics-foot-in-preference-to-postfix-syntax-t-foo

Reply:
Thanks @cartermp, I didn't know that